index.js

```diff
@@ -34,17 +35,20 @@ const getConfig = () => {
   const {api, clientId, secret} = config.paypal;
   if(!clientId) {
     throw new BedrockError(
-      'Missing PayPal clientId.', Errors.Data);
+      'Missing PayPal clientId. You can set this in ' +
+      'bedrock.config.paypal.clientId', Errors.Data);
```

Comment:
There are a number of reasons why I don't think we want this level of detail in an error message.  First and foremost is that it linking this error message to the bedrock config.  We usually don't have to concern ourselves with changing error messages if we add/change/remove bedrock configs stuffs.

Reply:
My major thinking on this was someone runs the tests, they get an error, this tells them exactly how to fix the error. @gannan08 opinion on this error message?

Reply:
I see why you did what you did @aljones15. Based off of the error message you are trying to make it clear how to resolve the error you're throwing. I do see @mattcollier's point as well in regards to changing error messages. What could be done here is an error message that says "the Variable X must be set in the bedrock config".

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/5/commits/91df6c5cd94b14bbc054dcb1fc97adc7c22162f4

@mattcollier  are you happy with this solution?

Reply:
Looks good, TY, resolved.